src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html

```diff
@@ -0,0 +1,113 @@
+<!---{
+ "preview": "amp4email",
+ "skipValidation": "true"
+}--->
+<!--
+## Introduction
+
+The AMPHTML Email format provides a subset of AMP components that you can use in email messages. Recipients of AMP emails can view and interact with the AMP components directly in the email message.
```

Comment:
Is AMPHTML Email is official term? Usually, `AMP HTML` is split into two words...